### PR TITLE
Adding SCIM attributes for admin group

### PR DESCRIPTION
**Pull request by ashensw**
Depends on https://github.com/wso2/carbon-kernel/pull/2778
Relates to https://github.com/wso2/product-is/issues/9040

### CODE REVIEW

File: components/org.wso2.carbon.identity.scim2.common/src/main/java/org/wso2/carbon/identity/scim2/common/utils/AdminAttributeUtil.java [hunk 1]
```diff
@@ -135,6 +136,23 @@ public static void updateAdminGroup(int tenantId) {
                         }
                         scimGroupHandler.addMandatoryAttributes(roleNameWithDomain);
                     }
+
+                    // Adding the SCIM attributes for admin group
+                    if (((AbstractUserStoreManager) userStoreManager).isRoleAndGroupSeparationEnabled()) {
+                        String adminGroupName = UserCoreUtil.removeDomainFromName(adminRoleName);
```
Comment (IsuraD) [inline components/org.wso2.carbon.identity.scim2.common/src/main/java/org/wso2/carbon/identity/scim2/common/utils/AdminAttributeUtil.java:142]: getAdminGroupName

**Review by IsuraD — APPROVED** [review]

File: components/org.wso2.carbon.identity.scim2.common/src/main/java/org/wso2/carbon/identity/scim2/common/utils/AdminAttributeUtil.java (hunk 1 above)
Comment (ashensw) [reply components/org.wso2.carbon.identity.scim2.common/src/main/java/org/wso2/carbon/identity/scim2/common/utils/AdminAttributeUtil.java]: Fixed and updated the PR.

**Comment by ashensw** [comment]
PR builder was successful with this PR: https://jenkins-support-all.wso2.com/jenkins/job/DEV_PR_BUILDER/821/

**Comment by ashensw** [comment]
PR builder was successful with this PR: https://jenkins-support-all.wso2.com/jenkins/job/DEV_PR_BUILDER/830

**Pull request merged**